setup.cfg

```diff
@@ -63,6 +63,7 @@ tests =
     types-toml==0.10.1
     # optional dependencies
     pywin32>=225; sys_platform == 'win32'
+    google-cloud-storage>=2.7.0
```

Comment:
This is only required for our (fake server) test fixture. gcsfs does not pin a required version, and dvc-gs doesn't require it for regular use